nilrt_snac/_Configs/_ConsoleConfig.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import subprocess
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+
+from nilrt_snac import logger
+
+
+class _ConsoleConfig(_BaseConfig):
+    def __init__(self):
+        pass  # Nothing to do for now
+
+    def configure(self, args: argparse.Namespace) -> None:
+        logger.info("Configuring console access...")
+        dry_run: bool = args.dry_run
+        if not dry_run:
+            logger.debug("Disabling console access")
+            subprocess.run(
+                ["nirtcfg", "--set", "section=systemsettings,token=consoleout.enabled,value=False"],
+                check=True,
+            )
+
+    def verify(self, args: argparse.Namespace) -> bool:
+        valid = True
+        result = subprocess.run(
+            ["nirtcfg", "--set", "section=systemsettings,token=consoleout.enabled,value=False"],
```

Comment:
Why not just `nirtcfg --get ...` here? I don't think `verify` should modify the system state.

Reply:
Yes, that was an error and is fixed.